agent/api/config_routes.py

```
@@ -83,4 +86,19 @@ async def delete_setting(
 
+    @router.get("/retention", response_model=DataRetentionConfig)
+    async def get_retention_policy(
+        retention_service: RetentionService = Depends(get_retention_service)
+    ):
+        """Get normalized data retention policy values."""
+        return retention_service.get_policy()
+
+    @router.post("/retention/cleanup", response_model=RetentionCleanupResponse)
+    async def run_retention_cleanup(
+        dry_run: bool = Query(default=True),
+        retention_service: RetentionService = Depends(get_retention_service)
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)